cmd/hope/kubectl.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+import (
+	"github.com/Eagerod/hope/cmd/hope/utils"
+	"github.com/Eagerod/hope/pkg/kubeutil"
+)
+
+var kubectlCmd = &cobra.Command{
```

Comment:
It almost feels cleaner to have this as part of the root command.

```
hope -- kubectl get pods
```

rather than

```
hope kubectl -- get pods
```